src/plays/airbnb-experiences-clone/AirbnbExperiencesClone.jsx

```diff
@@ -0,0 +1,21 @@
+import PlayHeader from 'common/playlists/PlayHeader';
+import './styles.css';
+import App from './App';
+
+// WARNING: Do not change the entry componenet name
+function AirbnbExperiencesClone(props) {
+  // Your Code Start below.
+
+  return (
+    <>
```

Comment:
If don't need can we remove the fragment?

Reply:
Yes, will do that.